packages/core-api/package.json

```diff
@@ -27,7 +27,7 @@
         "@arkecosystem/core-transactions": "4.0.0-next-1",
         "@arkecosystem/crypto": "4.0.0-next-1",
         "@hapi/boom": "9.1.4",
-        "@hapi/hapi": "20.1.5",
+        "@hapi/hapi": "20.2.1",
         "@hapi/hoek": "9.2.0",
```

Comment:
We strictly reference `@hapi/hoek@9.2.0` but `9.2.1` was also pulled in because `@hapi` packages reference `@hapi/hoek@9.x.x`.

I guess this kind of duplication will happen every time some strictly referenced package is updated.